.gitignore

```diff
@@ -4,6 +4,7 @@
 Temp/
 Obj/
 Bin/
+Builds/
```

Comment:
Note, I scoured the internet looking for standard output folders for builds and found that "Build" and "Builds" were by far the most common. I think we should adhere to that and so avoided the already supported Bin folder.